responses/__init__.py

```diff
@@ -1014,7 +1016,7 @@ def activate(
         func: Optional["_F"] = None,
         *,
         registry: Optional[Type[Any]] = None,
-        assert_all_requests_are_fired: bool = False,
+        assert_all_requests_are_fired: bool | None = None,
```

Comment:
```suggestion
        assert_all_requests_are_fired: Optional[bool] = None,
```

We need to use `Optional` for compatibility with older pythons.